src/components/clusterConfiguration/AdvancedNetworkFields.tsx

```diff
@@ -29,7 +34,7 @@ const AdvancedNetworkFields: React.FC = () => {
         min={clusterNetworkCidrPrefix}
         max={25}
```

Comment:
This will need to get adjusted based on whether the clusterNetworkCidr` is IPv4 or IPv6

Reply:
I don't think these are used anywhere... am I wrong ?

Reply:
min/max html attributes limit the number which user is able to input in the number type input